### PR TITLE
Options

### PR DESCRIPTION
**Pull request by g4s8**
zerocracy/datum#258 - options todo
(datum PR: https://github.com/zerocracy/datum/pull/309)

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#693](https://github.com/zerocracy/farm/issues/693) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#693](https://github.com/zerocracy/farm/issues/693) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/610863970). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/test/java/com/zerocracy/pmo/OptionsTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ */
+package com.zerocracy.pmo;
+
+/**
+ * Test case for {@link Options}.
+ * @author Yegor Bugayenko (yegor256@gmail.com)
+ * @version $Id$
+ * @since 0.21
+ * @todo #258:30min Options are not implemented. We should be able to
+ *  access `options/username.xml` file to read and modify user options.
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/pmo/OptionsTest.java]: @g4s8 I take it you mean that `username.xml` should be the name of the actual user's options? E.g. `options/carlosmiranda.xml` or `options/yegor256.xml`. I guess I'm a programmer but I initially interpreted `username.xml` to be literal, so perhaps we should be clearer.

**Comment by g4s8** [comment]
@carlosmiranda fixed, please see new commit

**Comment by carlosmiranda** [comment]
@yegor256 good to merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 please look at https://github.com/yegor256/rultor/issues/1250

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
The job [#693](https://github.com/zerocracy/farm/issues/693) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Pull request merged**